.travis.yml

```diff
@@ -7,11 +7,6 @@ os: osx
 osx_image: beta-xcode6.1
 
 env:
-  matrix:
-    
-    - CONDA_PY=27
-    - CONDA_PY=34
-    - CONDA_PY=35
```

Comment:
Ack! This doesn't look right. Could you please revert this @frol and raise an issue at [conda-smithy]( https://github.com/conda-forge/conda-smithy )?